include/odp/api/spec/ipsec.h

```diff
@@ -1097,6 +1195,18 @@ typedef struct odp_ipsec_sa_info_t {
  */
 int odp_ipsec_capability(odp_ipsec_capability_t *capa);
 
+/**
+ * Query capabilities for a given inline SA scope
+ *
+ * @param      scope  Inline SA scope
+ * @param[out] capa   Pointer to capability structure for output
+ *
+ * @retval 0 on success
+ * @retval <0 on failure
+ */
+int odp_ipsec_inline_capability(const odp_ipsec_inline_sa_scope_t *scope,
```

Comment:
This capability can be part of odp_pktio_capability_t ?

Reply:
This capability is associates with the SA scope, which consists of one _or more_ pktios and possibly look-a-side too. The same pktio could be used within different scopes. For instance pktio A could support HMAC-MD5 but pktio B not, then the capabilities for the scope that includes both pktio A and B cannot include HMAC-MD5.

Tying inline capabilities directly to pktios would not allow one to express what is supported when an SA state can be shared between multiple pktios (or a pktio and look-a-side).